dpdk/vr_dpdk_interface.c

```diff
@@ -392,8 +483,6 @@ dpdk_if_add(struct vr_interface *vif)
     } else if (vif_is_vhost(vif)) {
         return dpdk_vhost_if_add(vif);
     } else if (vif->vif_type == VIF_TYPE_AGENT) {
-        /* TODO: remove xconnect */
-        vhost_remove_xconnect();
```

Comment:
If the agent does not come up for some reason, we want packets from the physical interface to be cross-connected to vhost0. Why remove the xconnect call above?